src/patternfly/components/Content/content.scss

```diff
@@ -110,15 +109,16 @@
   pre,
   table {
     &:not(:last-child) {
-      // This variable name doesn't reflect the selector, it's an excpection to the variable system.
       margin-bottom: var(--pf-c-content--MarginBottom);
     }
   }
 
   ol,
   ul {
-    // This variable name doesn't reflect the selector, it's an excpection to the variable system.
     padding-left: var(--pf-c-content--ul--PaddingLeft);
+    margin: 0;
```

Comment:
could you write this as:
margin: var(--pf-c-content--ul--MarginTop) 0 0 var(--pf-c-content--ul--MarginLeft);

Reply:
Yep, good call. Also expanded the nesting CSS so you can nest an ol in a ul or vice versa and get proper nesting spacing. And that turned up a typo in the global css vars file for list-style!